spec/controllers/protected_resources_controller_spec.rb

```diff
@@ -297,4 +297,46 @@ def doorkeeper_forbidden_render_options(*)
       end
     end
   end
+
+  context 'when handle_auth_errors option is set to :raise' do
+    before do
+      config_is_set(:handle_auth_errors, :raise)
+    end
+
+    controller do
+      before_action :doorkeeper_authorize!
+
+      include ControllerActions
+    end
+
+    context 'and token is unknown' do
```

Comment:
I personally prefer to use always `when` with contexts. But it's totally up to you :)